eslint.config.mjs

```diff
@@ -1,8 +1,6 @@
-const jestPlugin = require('eslint-plugin-jest');
+import shareableConfig from './index.js';
```

Comment:
need explicit file extensions, see [it explained](https://www.totaltypescript.com/relative-import-paths-need-explicit-file-extensions-in-ecmascript-imports#the-error)

Reply:
relative import paths need explicit file extensions, yes - makes sense.